src/node_crypto.cc

```diff
@@ -2933,18 +2390,10 @@ int SSLWrap<Base>::TLSExtStatusCallback(SSL* s, void* arg) {
 
   if (w->is_client()) {
     // Incoming response
-    const unsigned char* resp;
-    int len = SSL_get_tlsext_status_ocsp_resp(s, &resp);
     Local<Value> arg;
-    if (resp == nullptr) {
-      arg = Null(env->isolate());
-    } else {
-      arg =
-          Buffer::Copy(env, reinterpret_cast<const char*>(resp), len)
-          .ToLocalChecked();
-    }
-
-    w->MakeCallback(env->onocspresponse_string(), 1, &arg);
+    MaybeLocal<Value> ret = GetSSLOCSPResponse(env, s, Null(env->isolate()));
+    if (ret.ToLocal(&arg))
```

Comment:
nit: inconsistent with code above that was using the `IsEmpty()`/`ToLocalChecked()` pattern. We might want to try to consistently use one or the other. I've no opinions on which would be preferable.

Reply:
There are still *many* areas of node_crypto.cc that can be improved, and I definitely plan to get to those, but this PR is already fairly large. I'd rather work on improving consistency incrementally.

Reply:
To be clear, I saw this as inconsistent not with _existing_ code (I wouldn't have brought that up, its too inconsistent already), but inconsistent with 2 places you _added_ code using IsEmpty()/ToLocalChecked() in this PR, so its all new code (or maybe I misread the diff and you were moving code around).

Reply:
ah, right. @addaleax what do you think?

Reply:
Using the IsEmpty/ToLocalChecked along with GetReturnValue().Set() is pretty common throughout the file. That could definitely go with a revamp. How about I work on cleaning those up in a separate PR. Using the ToLocal and FromMaybe patterns are a lot nicer overall, I think.

Reply:
>  Using the ToLocal and FromMaybe patterns are a lot nicer overall, I think.

I agree :+1:

If you want my personal opinion, I’d also inline the calls that lead to `MaybeLocal` values so that we don’t have duplicate variables, i.e.

```c++
if (GetSSLOCSPResponse(env, s, Null(env->isolate()))).ToLocal(&arg))
  ...
```

Reply:
Ok, I'll go ahead and do that for the new code, then will work on moving the existing code over in a separate PR